src/hooks/worlds/useWorldBySlug.ts

```diff
@@ -27,6 +27,7 @@ export const useWorldBySlug: (worldSlug?: string) => UseWorldBySlugResult = (
   const worldsRef = firestore
```

Comment:
Looks like duplicate. Probably should just remove it and rely on the function from `./api/world.ts`